rllib/env/wrappers/pettingzoo_env.py

```diff
@@ -188,37 +154,10 @@ def __init__(self, env):
         super().__init__()
         self.par_env = env
         self.par_env.reset()
+        self._agent_ids = self.par_env.agents
```

Comment:
`self._agent_ids` should be a set, so this line should be changed to:

```python
self._agent_ids = set(self.par_env.agents)
```

Reply:
done